modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -70,6 +77,7 @@ public static void testCleanup() {
     @Test
     public void test() throws SQLException {
         performSimpleTest(jdbcUrl);
+        performContainerStatusCheck(jdbcUrl);
```

Comment:
I would go with a separate test where you test both scenarios (with and without TC_DAEMON), just to make it clear if it's broken or not. IMO Parameterized runner seems to be misused in general in JDBCDriverTest :)
Thanks!